### PR TITLE
Component/CircularPie

### PR DESCRIPTION
**Pull request by christophschmitz**
Hello, I worked on my ProgressBar component.

- Install React-CircularProgressBar ([npm link](https://www.npmjs.com/package/react-circular-progressbar?activeTab=readme))
- Add Circular-Progressbar
- Add styling to ProgressBar
- Mock ProgressBar for storybook

This pull request resolves: #12 

[Deployment link](https://mastery-component-circu-z9fv54.herokuapp.com/storybook/?path=/story/component-progressbar--progressbar&globals=backgrounds.grid:false)

### CODE REVIEW

File: src/app/components/ProgressBar/ProgressBar.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+import styles from './ProgressBar.module.css';
+import './custom.css';
```
Comment (phlgr) [inline src/app/components/ProgressBar/ProgressBar.tsx:4]: This is needed for the loader?

**Review by phlgr — APPROVED** [review]
LGTM!

File: src/app/components/ProgressBar/ProgressBar.tsx (hunk 1 above)
Comment (christophschmitz) [reply src/app/components/ProgressBar/ProgressBar.tsx:4]: Yes, I need the custom.css to style the progress bar and the styles.module for the div around it. The size of the chart is determined by its parent div.

Comment (phlgr) [reply src/app/components/ProgressBar/ProgressBar.tsx:4]: Okay, feel free to merge

**Pull request merged**